cmake/InstallLayout.cmake

```diff
@@ -246,14 +246,21 @@ else()
   set(prerelease_string "-1")
 endif()
 
-
+#############
+# Filenames #
+#############
+set(unversioned_postfix "${PROJECT_VERSION}${prerelease_string}")
 # RPM filenames
-set(rpm-clients-filename "foundationdb-clients-${PROJECT_VERSION}${prerelease_string}")
-set(rpm-server-filename "foundationdb-server-${PROJECT_VERSION}${prerelease_string}")
+set(rpm-clients-filename "foundationdb-clients-${unversioned_postfix}")
+set(rpm-server-filename "foundationdb-server-${unversioned_postfix}")
+set(rpm-clients-versioned-filename "foundationdb${PROJECT_VERSION}-clients${prerelease_string}")
+set(rpm-server-versioned-filename "foundationdb${PROJECT_VERSION}-server${prerelease_string}")
 
 # Deb filenames
-set(deb-clients-filename "foundationdb-clients_${PROJECT_VERSION}${prerelease_string}")
-set(deb-server-filename "foundationdb-server_${PROJECT_VERSION}${prerelease_string}")
+set(deb-clients-filename "foundationdb-clients_${unversioned_postfix}")
```

Comment:
Is it okay that the unversioned deb filenames use an underscore while the rpm ones use a hyphen?

Reply:
Yes, I don't know why this is, but Alvin at the time insisted that this is necessary...